themes/openy_themes/openy_carnation/src/scss/modules/_global-search.scss

```diff
@@ -27,6 +28,38 @@
     }
 
     .form-actions {
+
+      display: block;
+      width: 100px;
+      height: 50px;
+      position: absolute;
+      right: 0;
+      top: 2px;
+
+      input {
+        width: 100%;
+        margin: 0;
+        height: 50px;
+        border-radius: 0 8px 8px 0;
+        background: #c6168d;
```

Comment:
same here, variables.scss. Leverage what we have for the sake of reusing. Reinventing the wheel is not a trend nowadays )))

Reply:
Fixed